homeassistant/components/lovelace/dashboard.py

```diff
@@ -251,10 +252,12 @@ async def _async_load_data(self) -> collection.SerializedStorageCollection | Non
 
     async def _process_create_data(self, data: dict) -> dict:
         """Validate the config is valid."""
-        if "-" not in data[CONF_URL_PATH]:
+        url_path = data[CONF_URL_PATH]
+
+        if url_path != "map" and "-" not in url_path:
```

Comment:
I was chatting with @piitaya about this, an alternative to allowing `map` here would be to have a hard coded redirect in frontend, but that seems like a surprising behavior to me.

Reply:
Can we add a force parameter to not check this rule? So we can bypass this rule for the map and iframe migration?